src/components/Header/header.scss

```diff
@@ -88,10 +89,10 @@
   padding: 0;
   margin: 0;
   padding: 13px 15px;
-  
-    @include on-desktop {
-      padding: 21px 23px;
-    }
+
+  @include on-desktop {
+    padding: 21px 23px;
+  }
```

Comment:
Why we use padding size 13 / 15 instead of  12 / 16
Usually size should devide to 4